README.md

```diff
@@ -74,10 +74,21 @@ source venv/bin/activate
 
 ### Install with pip
 
+```bash
+pipenv install --python 3
 ```
-python3 -m pip install -r requirements.txt
+
+### Install with pipenv
+
+```bash
+pipenv run python -m unittest discover
```

Comment:
This looks duplicate of below. 
Seems like this should be 
`python3 -m pip install -r requirements.txt` ?
or `pip3 install -r requirements.txt`

Reply:
Done.